### PR TITLE
feat(librispeech): add working jax code

### PR DESCRIPTION
**Pull request by ClashLuke**
The training is still going, but it looks pretty good so far.\
As discussed, the Jax version adds significant padding (which isn't there in the PyTorch version), which should cause it to run 2x to 16x slower. At the same time, I had to add masks to BatchNorm, which will cause the performance of this model to deviate slightly from that of the PyTorch model.
There were also many issues with the original Jax submission, so we should probably double-check with the PyTorch version and avoid these in the Conformer implementation.

grafik

### CODE REVIEW

**Comment by ClashLuke** [comment]
We now have [preliminary logs](https://github.com/mlcommons/algorithmic-efficiency/files/8459994/baseline.txt) showing the poor stability of our model: cer

**Comment by ClashLuke** [comment]
```
python3 -m pip install --upgrade pip
python3 -m pip install --upgrade "jax[tpu]" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
python3 -m pip install flax

**Comment by znado** [comment]
cleaning up old PRs, closing
